tmk_core/common/action.c

```diff
@@ -847,6 +847,18 @@ void unregister_code(uint8_t code)
     #endif
 }
 
+/** \brief Utilities for actions. (FIXME: Needs better description)
+ *
+ * FIXME: Needs documentation.
+ */
+void tap_code(uint8_t code) {
+  register_code(code);
+  #if defined(TAP_CODE_DELAY) && TAP_CODE_DELAY > 0
```

Comment:
```suggestion
#if TAP_CODE_DELAY > 0
```
See above. Also, I'd change the preprocessor directive indentation to match the rest of this file.